cpp/daal/src/services/library_version_info.cpp

```diff
@@ -25,6 +25,7 @@
 #include "src/services/daal_version.h"
 #include "src/services/service_defines.h"
 #include "services/env_detect.h"
+#include "services/daal_defines.h"
 #include "mkl_daal.h"
```

Comment:
Why `mkl_daal.h` is required here?

Reply:
`fpk_serv_cpuisknm` variable defined in `mkl_daal.h`
https://github.com/oneapi-src/oneDAL/blob/master/cpp/daal/src/services/library_version_info.cpp#L48